server/lib/src/macros.rs

```diff
@@ -119,16 +119,22 @@ macro_rules! run_create_test {
     }};
 }
 
-// #[macro_export]
 #[cfg(test)]
+/// Runs a test with preloaded entries, then modifies based on a filter/list, then runs a given check
```

Comment:
Funny, as I'm about to try to kill these macros. They're blocking us from removing async-std.